pkg/operator/certrotation/event.go

```diff
@@ -0,0 +1,23 @@
+package certrotation
+
+import (
+	"fmt"
+
+	"github.com/openshift/library-go/pkg/operator/events"
+	"github.com/openshift/library-go/pkg/operator/resource/resourcehelper"
+	"k8s.io/apimachinery/pkg/runtime"
+)
+
+func emitEvent(recorder events.Recorder, action string, obj runtime.Object, err error) {
```

Comment:
Instead of this new function I would prefer to reuse [reportCreateEvent](https://github.com/openshift/library-go/blob/6ca0621840e45bed8686978644512dc3320fe5c7/pkg/operator/resource/resourceapply/event_helpers.go#L25C6-L25C23) and [reportUpdateEvent](https://github.com/openshift/library-go/blob/6ca0621840e45bed8686978644512dc3320fe5c7/pkg/operator/resource/resourceapply/event_helpers.go#L34C6-L34C23).

I think we could open a new PR in which we would:
1. crate a new `helpers` pkg in `library-go/pkg/operator/resource/helpers`
2. we would move `operator/resource/resourceapply/event_helpers.go` to the new pkg (with tests)
3. we would export all `report` methods.

Then we could remove this file and use the new exported methods directly. WDYT?